common/scripts/update-npm-dependencies.js

```diff
@@ -0,0 +1,141 @@
+const ncu = require('npm-check-updates');
+const Octokit = require('@octokit/rest');
+const { installAndRun, findRushJsonFolder, RUSH_JSON_FILENAME } = require("./install-run");
+const fs = require('fs');
+const { join } = require('path');
+const { promisify } = require('util');
+const { spawnSync } = require('child_process');
+const readdir = promisify(fs.readdir);
+const exists = promisify(fs.exists);
```

Comment:
Similarly, can you use `existsSync`? `exists` is deprecated due to not having error first callback, and the promisify only works due to `util.promisify.custom` which changes the behavior.

Reply:
changed